README.md

```diff
@@ -0,0 +1,204 @@
+<a name="readme-top"></a>
+<div align="center">  
+  <h2><b>Air Metrics WebApp</b></h2>
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+<!-- - [❓ FAQ (OPTIONAL)](#faq) -->
+- [📝 License](#license)
+
+<br>
+
+# 📖 Math Magicians <a name="API-based webapp"></a>
+>"Air Metrics" a Single Page App (SPA) that allows users to:
+>- Display air pollution data.
+>- Weather Data.
+<br>
+
+## 🛠 Built With <a name="built-with"></a>
+- React
+- Redux
+- SASS
+- Linters
+
+### Tech Stack <a name="tech-stack"></a>
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://create-react-app.dev/docs/getting-started/">React</a></li>
+    <li><a href="https://redux.js.org/">Redux</a></li>
+    <!-- <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://webpack.js.org/guides/getting-started/#basic-setup">Webpack</a></li>
+    <li><a href="https://jestjs.io/docs/getting-started">Jest</a></li> -->
+  </ul>
+</details>
+<details>
+  <summary>API</summary>
+  <ul>
+    <li><a href="https://ipgeolocation.io/">IP Geolocalistion</a></li>
+    <li><a href="https://openweathermap.org/weather-conditions">Weather Data</a></li>
+    <li><a href="https://openweathermap.org/api/air-pollution">Pollution Data</a></li>
+  </ul>
+</details>
+<br>
+
+### Key Features <a name="key-features"></a>
+- A home page showing User city data and closest cities data
+- Details page showing details informations about a given location
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+>[Live Demo](https://airmetrics.onrender.com)
```

Comment:
## VIDEO PRESENTATION

- Hey, great job adding the live demo version, however, you're yet to add the missing video presentation which is a key feature in this project to be approved. kindly see below the instructions to implement it below:

![image](https://github.com/momo-87/air-metrics-webapp/assets/59397159/d0da290f-a06e-48a6-9f83-782ca18b5931)

Reply:
Thank you, I really appreciate the time you spend reviewing my project.
The video presentation link was added to the readme file.